src/nvim/main.c

```diff
@@ -1398,6 +1398,17 @@ static void load_plugins(void)
 }
 
 /*
+ * Read all the early plugin files (those are sourced before the vimrc)
+ */
+static void load_early_plugins(void)
```

Comment:
I think it's fine currently; the benefit of having a separate interface outweighs the duplication, IMO.